### PR TITLE
[MP-1591] Display status of merchant branch rebase and tests on PR

### PR DESCRIPTION
**Pull request by VincentGalloroBolt**
#changelog [MP-1591] Display status of merchant branch rebase and tests on PR

Show the status of rebasing and unit tests on merchant branches in the current PR

### CODE REVIEW

**Comment by usmanismail-bolt** [comment]
The tests passed but there was an error in the script

```
./Test/scripts/ci-unit.sh: line 23: TEST_ENV: unbound variable
++ curl -s https://bolt-devops.s3-us-west-2.amazonaws.com/testing/codecov_uploader
unit tests failed
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -129,10 +129,11 @@ jobs:
 
   rebase-and-unit-test:
     docker:
-      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v2
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v1
```
Comment (usmanismail-bolt) [inline .circleci/config.yml:132]: whats the difference between v2 and V1 here?
Why downgrade?

**Comment by usmanismail-bolt** [comment]
Can we add another branch [here](https://github.com/BoltApp/bolt-magento2/blob/vincent/circleci/display-rebase-status-on-pr/.circleci/scripts/auto-rebase-branches.txt) to make double sure that multiple branches works correctly and what it looks like in the output.

File: .circleci/config.yml (hunk 1 above)
Comment (VincentGalloroBolt) [reply .circleci/config.yml:132]: This is required, otherwise memory limit breaks for some reason

**Pull request merged**